starboard/android/shared/player_get_preferred_output_mode.cc

```diff
@@ -82,7 +82,8 @@ SbPlayerOutputMode SbPlayerGetPreferredOutputMode(
         is_sdr) {
       return kSbPlayerOutputModeDecodeToTexture;
     }
-    SB_LOG(ERROR) << "Invalid output mode due to play HDR video on sub player";
+    SB_LOG_IF(ERROR, !is_sdr)
+        << "Invalid output mode due to HDR videos (decode-to-texture).";
```

Comment:
"Returning kSbPlayerOutputModeInvalid as HDR videos aren't supported under decode-to-texture." (feel free to rephrase)

Reply:
Done.